lib/commands/install.js

```diff
@@ -56,12 +58,16 @@ exports.usage = '' +
 exports.run = function (settings, args) {
     var a = argParse(args, {
         'repository': {match: '--repository', value: true},
-        'target_dir': {match: '--package-dir', value: true}
+        'target_dir': {match: '--package-dir', value: true},
+        'source_dir': {match: '--source-dir', value: true}
     });
 
     var opt = a.options;
     var pkg = a.positional[0] || '.';
 
+    if (opt.source_dir) {
+        opt.source_dir = opt.source_dir.split(';').reverse();
```

Comment:
Cool, I think colon is the natural fit since like you say we're mostly unix focused right now.